posix.mak

```diff
@@ -161,6 +165,14 @@ $(DOC_OUTPUT_DIR)/%.verbatim : %.dd verbatim.ddoc $(DMD)
 $(DOC_OUTPUT_DIR)/%.php : %.php.dd $(DDOC) $(DMD)
 	$(DMD) -c -o- -Df$@ $(DDOC) $<
 
+$(DOC_OUTPUT_DIR)/css/% : css/%
+	@mkdir -p $(dir $@)
+ifeq (1,$(CSS_MINIFY))
+	curl -X POST -s --data-urlencode 'input@$<' http://cssminifier.com/raw >$@
```

Comment:
Is there some local program we could settle on, e.g. yuglify.
Make sure to include -f and -S (-fsS) as curl arguments.